desktop/npm-helper.js

```diff
@@ -333,26 +341,135 @@ function undiff () {
   console.log('Success! Wrote ./log.json')
 }
 
-function updatedFonts () {
-  const project = JSON.parse(fs.readFileSync('./shared/images/iconfont/kb-icomoon-project.json', 'utf8'))
-  const fontPrefix = 'kb-iconfont-'
-  const icons = {}
+function svgToGridMap () {
```

Comment:
This analyzes our svgs and divides them into the grid sizes for later consumption